src/main.rs

```diff
@@ -100,32 +116,27 @@ fn main() -> sysexits::ExitCode {
         None => 3,
         Some(Ok(it)) => it,
         Some(Err(_)) => {
-            eprintln!("duration must be an integer value!");
+            eprintln!("Duration must be an integer value!");
```

Comment:
I found this typo by accident and decided to fix it.  The message will be written to `stderr` as a standalone phrase, so it will need a capital letter at its beginning.